var/spack/repos/builtin/packages/hoomd-blue/package.py

```diff
@@ -51,17 +51,23 @@ class HoomdBlue(CMakePackage):
     extends('python')
     depends_on('python@2.7:')
     depends_on('py-numpy@1.7:', type=('build', 'run'))
-    depends_on('cmake@2.8.0:', type='build')
+    depends_on('cmake@2.8.0:3.9.6', type='build')
     depends_on('pkgconfig', type='build')
     depends_on('mpi', when='+mpi')
     depends_on('cuda@7.0:', when='+cuda')
     depends_on('doxygen@1.8.5:', when='+doc', type='build')
 
     def cmake_args(self):
         spec = self.spec
+        python_maj, python_min = (
+            str(spec['python'].version.dotted).split('.')[:2]
+        )
+        install_path = '{0}/lib/python{1}.{2}/site-packages'.format(
```

Comment:
@snehring the Spack `python` package includes a `site_packages_dir` property which should be able to calculate this automatically for you. Although what you have looks correct, would you mind updating this:

```
install_path = spec['python'].package.site_packages_dir
```

Reply:
Done. I figured there was something like that built in. Didn't look close enough.